cfme/tests/test_replication.py

```diff
@@ -322,7 +337,22 @@ def test_replication_global_to_remote_new_vm_from_template():
             2.
             3. VM created in the Remote, no errors.
     """
-    pass
+    remote_app, global_app = setup_replication
+    remote_provider = provider_app_crud(RHEVMProvider, remote_app)
+    remote_provider.setup()
+    assert remote_provider.name in remote_app.managed_provider_names, "Provider is not available."
+
+    new_vm_name = "test_replication_global_to_remote_new_vm_from_template"
+    global_provider = provider_app_crud(RHEVMProvider, global_app)
+    vm = create_vm(provider=global_provider, vm_name=new_vm_name)
+    remote_provider.refresh_provider_relationships()
+
+    vmObj_list = remote_app.collections.infra_vms.filter({"provider": remote_provider}).all()
+    all_vm_names = [vmObj.name for vmObj in vmObj_list]
+    assert new_vm_name in all_vm_names, f"{new_vm_name} vm is not found in Remote Appliance"
+
+    # CleanUp
+    vm.cleanup_on_provider()
```

Comment:
This should be a part of cleanup, `request.addfinalizer(vm.cleanup_on_provider)`